---
title: "Curriculum"
date: 2024/01/05 10:22
---

Since 2022: consulting developer, IBM France
--------------------------------------------

**Design and developement of cloud services**

Company
: [IBM Consulting France](https://www.ibm.com/fr-fr/consulting)


2020-2022: Starting Research Position (SRP), Inria Saclay (France)
--------------------------------------

**Abstractions of influence networks**

Group
: [Lifeware](http://lifeware.inria.fr)

Institute
: Inria Saclay - Île-de-France





2019-2020: Visiting Scientist, FU Berlin (Germany)
--------------------------------------

**Trapspaces in buffered networks**

Group
: [Discrete Biomathematics](https://www.mi.fu-berlin.de/en/math/groups/dibimath)

University
: Freie Universität Berlin 



2017-2019: Researcher, IBENS (France)
--------------------------------------

**Modelling heterogeneity in immune cells**

Group
: [Computational Systems Biology](http://www.ibens.ens.fr/spip.php?rubrique27)

Lab
: IBENS (ENS - CNRS UMR8197 - INSERM U1024)

Main results:

* CoLoMoTo notebook: a reproducible integrative platform for qualitative modelling
* In-silico design of reprogramming strategies for Th cells
* Ongoing work on models of T cells and dendritic cells



2015-2017: Postdoc, Université de Montpellier (France)
---------------------------------------------------------


**Signalling networks in cancer progression**

Group
: [Biological Physics and Systems Biology](https://systems-biology-lphi.cnrs.fr/)

Lab
: [DIMNP (now LPHI - UMR 5235)](https://lphi.umontpellier.fr/)

University
: [Université de Montpellier](http://www.umontpellier.fr)


Main results:

* Syk and PTPL1 phosphorylation networks in breast cancer cells
* Timescale-based reduction of quantitative networks


2010-2015:    Postdoc Université de Lausanne (Switzerland)
----------------------------------------------------------

**Genomic regulations in metabolism**

Group
: Béatrice Desvergne (now retired)

Institute
:  [Center for Integrative Genomics](http://www.genomyx.ch)

University
: [Université de Lausanne](http://www.unil.ch)


Main results:

* Genome-wide study of nuclear receptors functional interactions in HepaRG cells
* Genome-Wide Analysis of SREBP1 Activity around the Clock in mouse liver cells
* Epigenetics of early aging in mouse adipose tissue



2006-2009: PhD, Université de la méditeranée, Marseille (France)
----------------------------------------------------------------

**Logical model of the differentiation of T helper cells** - [thesis (fr)](/these_AurelienNaldi_2009.pdf)

Lab
: [TAGC (INSERM U1090)](http://tagc.univ-amu.fr)

University
: [Université de la méditerranée](http://www.univ-amu.fr)

Supervisers
: [Denis Thieffry](http://www.ibens.ens.fr/spip.php?rubrique27)
  and [Claudine Chaouiya](http://claudine.chaouiya.perso.luminy.univ-amu.fr/)


Main results:

* Formal methods for the reduction and analysis of logical models
* GINsim: a software tool for the definition and analysis of logical models
* Diversity and plasticity of Th cell types predicted from regulatory network modelling




Education, Marseille (France)
-----------------------------

2006
: MSc in [Bioinformatics (BBSG)](http://biologie.univ-mrs.fr/masterBBSG/>)

2004
: MSc in [Computer science (DESS Informatique)](http://masterinfo.univ-mrs.fr)

1999
: DEUG in life sciences



Skills
------


Languages
: French (native), English (fluent), German (basics)

Programming
: Fluent in Java and python. Worked with C/C++, Rust, R, javascript, php, bash

Bioinformatics
: Discrete modeling, graph visualisation, Genomics (microarray, ChIP-seq), Proteomics (SILAC)

Biology
: Applications to Cell cycle, Immune system, Metabolism, Cancer

Informatics
: Linux administration, git, SQL, LaTeX, Docbook

